lib/tables.dart

```diff
@@ -28,7 +29,8 @@ class Table<T> extends Object with SetStateMixin {
     _spacerBeforeVisibleRows = new CoreElement('tr');
     _spacerAfterVisibleRows = new CoreElement('tr');
 
-    _visibilityObserver = new IntersectionObserver(_visibilityChange);
+    // TODO(jacobr): remove call to allowInterop once bug in dart:html is fixed.
```

Comment:
Is there a GitHub issue for this bug? If so, adding it in a comment would make it easier to check when it can be removed.

Reply:
For browser support I think we only need to support chrome as we will generally be launching this app directly from the command line. Added
https://github.com/flutter/devtools/issues/66
to remind users if they open the devtools with an unsupported browser.

Filed
https://github.com/dart-lang/sdk/issues/35484
to fix dart:html and referenced that bug in my comment.

Reply:
The tests for this do run on travis as they broke without the allowInterop call.

Reply:
Ah, cool - I couldn't remember which ones we'd got working :-)